BrainPortal/lib/cbrain_task_generators/templates/portal.rb.erb

```diff
@@ -412,7 +415,15 @@ class CbrainTask::<%= name %> < <%= (descriptor['custom'] || {})['cbrain:inherit
       params_errors.add(grpName[group], errMsg) unless hasOneActive.(group)
     end
 % end
-%#
+% if hasAllNone
+    # Lambda for checking all-or-none (either none or all are active)
+    hasAllNone = lambda { |gid| gidToMbrs[gid].select{ |m| ! isInactive.(m.to_sym) }.count == 0 ||
+                           gidToMbrs[gid].select{ |m| isInactive.(m.to_sym) }.count == 0}
```

Comment:
I'm not going to request a change but this complex test could be made way simpler.

For instance:

Using `.all?` instead of `.select { code }.count == 0`

```ruby
asAllNone = lambda { |gid| gidToMbrs[gid].all? { |m|  ! isInactive.(m.to_sym) } ||
                                            gidToMbrs[gid].all?{ |m|  isInactive.(m.to_sym) }
```

Or you could just make sure all the `isInactive()` call return the same thing:

```
hasAllNone = lambda { |gid| gidToMbrs[gid].map { |m| isInactive.(m.to_sym) }.uniq.count == 1 }
```

Reply:
These are cool, especially the second one. Perhaps too cool, as requires some thinking to understand.